crates/jmux-proto/src/lib.rs

```diff
@@ -2,12 +2,12 @@
 //!
 //! [source]: https://github.com/Devolutions/devolutions-gateway/blob/master/docs/JMUX-spec.md
 
-use bytes::{Buf as _, BufMut as _};
 use core::fmt;
-use smol_str::SmolStr;
 
+use bytes::{Buf as _, BufMut as _};
 // We re-export these types, because they are used in the public API.
 pub use bytes::{Bytes, BytesMut};
+use smol_str::SmolStr;
```

Comment:
note: I don’t really like the mixed `pub use` and `use` like that. We use `#[rustfmt::skip]` in IronRDP. I wonder if there is an option for that today?